streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationBackUpService.java

```diff
@@ -74,7 +74,7 @@ public interface ApplicationBackUpService extends IService<ApplicationBackUp> {
    *
    * @param appParam the application to be removed
    */
-  void removeApp(Application appParam);
+  void removeByApp(Application appParam);
```

Comment:
Some places are Boolean remove (Application appParam); Could you please check it out?